ARGO/include/Utilities.h

```diff
@@ -28,9 +28,16 @@ namespace Utilities
 	const int LEVEL_TILE_WIDTH = 30;
 	const int LEVEL_TILE_HEIGHT = 20;
 	const int TILE_SIZE = 64;
+
 	const int PLAYER_RADIUS = 32;
+	const int PLAYER_FIRE_RATE = 1000 / 10;
```

Comment:
why not just 100 here

Reply:
Milliseconds divided by how many times per seconds you want to fire, makes more sense for changing it in the future